### DISCUSSION TITLE
Splash notes from week 7 upd

### DISCUSSION THREAD

**Discussion by LOLZSICKA** [Saltssaumure/fnf-osu-mania-skin#38] [Ideas]
i realised if voez (and a few other skins) have custom hit effects, that means this skin here can have the splash effects from the real game

**Comment by Saltssaumure** [comment]
Could you provide a screenshot of the splash effects from FNF that you'd like added?

**Reply by LOLZSICKA** [reply]
i just realised the one thing i found out about can only take like one color and applies to all columns so unless its fine being gray or smthn its not really gonna work that well as i thought it would (not unless osu then decides to just make it so note hit effect can be seperate colors) and also i was talking about the splash effect for friday night funkin that got added in the week 7 update

**Reply by RhythMark** [reply]
![download](https://user-images.githubusercontent.com/102244928/160175805-c45cc9a1-f0d0-4df5-92d9-f488f0c12bbd.png)
